packages/menu-bar/src/vaadin-menu-bar-interactions-mixin.js

```diff
@@ -36,6 +36,9 @@ export const InteractionsMixin = (superClass) =>
 
       this.addEventListener('keydown', (e) => this._onKeydown(e));
       this.addEventListener('focusin', (e) => this._onFocusin(e));
+      this.addEventListener('focusout', (e) => this._onFocusout(e));
```

Comment:
Consider using `FocusMixin` which brings `focusin` and `focusout` listeners, and then override `_setFocused` method (maybe without calling `super._setFocused` as we don't need `focused` attribute).

Reply:
Done